wordpress-mobile/WordPress-iOS
# Remove LoginEpilogue

When you log in, the app no longer asks you to select a site after login and defaults to either your primary or the most recently created site. This behavior is consistent with wp.com.

This change was implemented in the scope of the Site Picker redesign. It removes the deprecated `BlogListDataSource` and `WPBlogTableViewCell` usage. 

Follow-up: https://github.com/wordpress-mobile/WordPress-iOS/pull/23470 (Integrate TipKit) where we teach the user how to use the standard site picker instead of showing this ad-hoc one that you never see again and that doesn't explain where to select sites later.

<img width="320" alt="Screenshot 2024-08-01 at 3 37 41 PM" src="https://github.com/user-attachments/assets/e9bbc1a9-7ff0-4231-9fb7-6312cce4def8">


## To test:

- Login with an existing account that has multiple sites and a primary site
- Verify that the app automatically opens your primary site

## Regression Notes
1. Potential unintended areas of impact


2. What I did to test those areas of impact (or what existing automated tests I relied on)


3. What automated tests I added (or what prevented me from doing so)

PR submission checklist:


Testing checklist:
